spa/src/pages/Connections/index.tsx

```diff
@@ -0,0 +1,61 @@
+import { useEffect, useState } from "react";
+import ApiRequest from "../../api";
+import SyncHeader from "../../components/SyncHeader";
+import Step from "../../components/Step";
+import { Wrapper } from "../../common/Wrapper";
+import GitHubCloudConnections from "./GHCloudConnections";
+import GitHubEnterpriseConnections from "./GHEnterpriseConnections";
+import { GHSubscriptions } from "../../rest-interfaces";
+import { reportError } from "../../utils";
+import SkeletonForLoading from "./SkeletonForLoading";
+import { useNavigate } from "react-router-dom";
+
+const Connections = () => {
+	const navigate = useNavigate();
+	const [isLoading, setIsLoading] = useState<boolean>(false);
+	const [ghSubscriptions, setSubscriptions] = useState<GHSubscriptions | null>(null);
+	const fetchGHSubscriptions = async () => {
+		try {
+			setIsLoading(true);
+			const { data } = await ApiRequest.subscriptions.getSubscriptions();
+			setSubscriptions(data);
+		} catch (e) {
+			reportError(e, { path: "Fetching subscriptions" });
+		} finally {
+			setIsLoading(false);
+		}
+	};
+	useEffect(() => {
+		fetchGHSubscriptions();
+	}, []);
+
+	// If there are no connections then go back to the start page
+	useEffect(() => {
+		if (!ghSubscriptions?.ghCloudSubscriptions && ghSubscriptions?.ghEnterpriseServers && ghSubscriptions.ghEnterpriseServers?.length === 0) {
+			navigate("/spa");
```

Comment:
I think we should add a ticket to update this route

Reply:
Will address that in the future, when we have proper design for this whole flow.